parm/aqm.rc

```diff
@@ -322,6 +322,80 @@ fire_species::
     FRP     1000000    MeanFRP     1
 ::
 
+#
+# PT Source
+#
```

Comment:
Don't we need a flag for PT source like `do_aqm_pt_source` as others?

Reply:
again sorry i confused this with the separate PR.  Removing this section (it would not read it as you noted but lets keep this PR clean)